tools/vsextension/ProjectTemplates/oeenclave-linux/MyEnclave.edl

```diff
@@ -0,0 +1,11 @@
+enclave {
+    trusted {
+        /* define ECALLs here. */
+        public int ecall_DoWorkInEnclave();
```

Comment:
The naming convention of the ecall an ocall function seems out of place considering rest of the code generated by the wizard.